org.mwc.cmap.legacy/src/MWC/GUI/Plottables.java

```diff
@@ -347,15 +347,8 @@ public synchronized void paint(final CanvasType dest)
 	 */
 	public MWC.GenericData.WorldArea getBounds()
 	{
-		// do we need to recalculate?
-		if (_myArea == null)
-		{
-			// yup, get on with it...
-			_myArea = recalculateAreaCovered();
-		}
-
-		return _myArea;
-
+		// yup, get on with it...
+		return recalculateAreaCovered();
 	}

```

Comment:
so, @saulhidalgoaular  - shouldn't we remove the `_myArea` field?

Reply:
oh yes. 1 min